### ISSUE TITLE
i am setting up this in Heroku server but not working

### ISSUE THREAD

**Issue by priyanshurohilla** [footniko/mediasoup-sample#12]
I am trying to deploy this webapp in heroku server. It is working on localhost ,I can see the different webcam elements but in heroku webcam element is not being displayed. Can you help me out? 
 You can refer to this image:

![WhatsApp Image 2020-05-24 at 1 42 42 AM](https://user-images.githubusercontent.com/56466692/82739814-814dc400-9d60-11ea-99d3-d8dfbd4656c2.jpeg)

**Comment by footniko** [comment]
@priyanshurohilla make sure you've set the **rtcAnnouncedIPv4** of the server config to a public IP of your server.

**Comment by priyanshurohilla** [comment]
how do i find that `rtcAnnouncedIPv4` of heroku server?

**Comment by footniko** [comment]
@priyanshurohilla it's not on Heroku server, it's in **server/config.js** file of this project.

**Comment by priyanshurohilla** [comment]
yes i know that, what i am saying is how do i find the public IP of Heroku server to be written in that `config.js` file?

**Comment by footniko** [comment]
I do not use Heroku, pls google it.
One of the options:
```
ping yourherokudomain
```

**Comment by priyanshurohilla** [comment]
ok cool!! i already did this, but still the same error is coming?🤔

**Comment by priyanshurohilla** [comment]
Basically what am i able to see is that if i change the video src to a public available video.. Then it works perfectly. I can see that video, but normally can't see the live feed!! Is there any problem with the streaming of video?

**Comment by footniko** [comment]
Try to set the IP to a local IP of the server. SSH to your Heroku's server and check it or check it in the control panel if it's possible.

**Comment by samaprk** [comment]
hi, I am also having the same problem, I changed the config.js rtcannounceipv4 to the Heroku domain name of the app i created but still the problem remains. What do you mean by setting the IP to local IP? Do you mean that I should pick one from the range of IP given by nslookup? 
I am not a network person. Would be thankful if you help me

**Comment by footniko** [comment]
@samaprk 
>I changed the config.js rtcannounceipv4 to the Heroku domain name of the app i created but still the problem remains.

You have to set IP, not the domain name. Also, make sure the following range of UDP ports is open:
```
rtcMinPort: 40000,
rtcMaxPort: 49999,
```
